CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* feat: Implement 'Uint512DivMod' hint [#1171](https://github.com/lambdaclass/cairo-rs/pull/1171)
```

Comment:
just a detail, the implementation of the cairo 1 hints are mentioned above in the changelog.
Just add the link to this PR in that line and remove this line